generator/src/generator.rs

```diff
@@ -223,10 +223,17 @@ fn generate_enum(
     });
 
     let grammar_doc = &doc_comment.grammar_doc;
-    let mut result = quote! {
-        #[doc = #grammar_doc]
-        #[allow(dead_code, non_camel_case_types, clippy::upper_case_acronyms)]
-        #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+    let mut result = if grammar_doc.is_empty() {
+        quote! {
+            #[allow(dead_code, non_camel_case_types, clippy::upper_case_acronyms)]
```

Comment:
I think it's optional. `missing_docs` is allowed by default. If user really wants to turn on the lint, it will required to add docstring in the grammar file.